server/src/userManagement.mjs

```diff
@@ -84,9 +84,13 @@ function getWssForUser(userId) {
   return undefined;
 }
 
+//get the ws for user,if user has ws already,it will return the latest ws connection
 function getWsForUser(userId) {
+  let wsArray = [];
   if ( user2ws.has(''+userId) ) {
-    return user2ws.get(''+userId);
+    wsArray=user2ws.get(''+userId);
+    let latestWsConnection=wsArray[(wsArray.length-1)]
```

Comment:
Can/should use 'const' here.

Reply:
More importantly, is this the right "logic"? Do we always want to return the last ws object in the array? Is there going to be code that removes items from this array as connections disconnect? It feels like there's something missing here.

Reply:
Declared wsArray as const.

Reply:
As per the ask,changes should support the backward compatibility, which has to return the last ws object in the array.I'm suspecting user2ws.delete() will delete the user along with the list of ws connections in the array once connection disconnect.

Reply:
Hmm... const should be allowed when the value of a variable doesn't change, but it's okay with let.